docs/extensions/resources/controlplane-exposure.md

```diff
@@ -2,6 +2,8 @@
 title: ControlPlane Exposure
 ---
 
+> **⚠️ Disclaimer:** The `ControlPlane` resource with purpose `exposure` is deprecated and will be removed in a future Gardener release. Since the enablement of SNI, the `exposure` purpose is no longer used.
```

Comment:
I know that I suggested you the deprecation message from https://github.com/gardener/gardener/pull/11372/files. 🙈 
However, now going over it again, we could make use of the [new GitHub syntax for notes](https://github.com/orgs/community/discussions/16925):

```suggestion
> [!WARNING]
> The `ControlPlane` resource with purpose `exposure` is deprecated and will be removed in Gardener v1.123. Since the enablement of SNI, the `exposure` purpose is no longer used.
```